.gitignore

```diff
@@ -7,7 +7,8 @@ build/
 #Hardhat files
 cache
 artifacts
-typechain
+/typechain
+/src
```

Comment:
Opinions on folder naming for the package?

Reply:
Post some options?